core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -110,68 +123,51 @@ export class PreferencesPageComponent {
     return this.urlInterpolationService.getStaticImageUrl(imagePath);
   }
 
-  private _saveDataItem(
-      updateType: string,
-      data: string | string[] | EmailPreferencesBackendDict
-  ): void {
+  handleBioChange(): void {
+    this.hasUnsavedChanges = true;
+    this.unsavedChanges.user_bio = true;
```

Comment:
Here we are setting unsavedChnages.'user_bio' to true, so that we can filter out the update_type in savePreferences function below.
I will change the names as specified.